owncloudData/src/androidTest/java/com/owncloud/android/data/user/UserDaoTest.kt

```diff
@@ -57,23 +57,23 @@ class UserDaoTest {
 
     @Test
     fun insertQuotaForAccount() {
-        userDao.insert(userQuotaMapper.toEntity(OC_USER_QUOTA)!!.copy(accountName = OC_ACCOUNT_NAME))
+        userDao.insert(OC_USER_QUOTA.toEntity(OC_ACCOUNT_NAME))
```

Comment:
You could also have `OC_USER_QUOTA_ENTITY` somewhere and call `OC_USER_QUOTA.toEntity` from there 🍻

Reply:
Ok, applied 👍